docs/Scripts/logmonitor.md

```diff
@@ -14,4 +14,31 @@ Use the `deploy-as-systemd.sh` script to create a systemd unit file (deployed to
 Log output is handled by syslog and end up in the systems standard syslog file, normally `/var/log/syslog`. `journalctl -f -u cnode-logmonitor.service` can be used to check service output(follow mode). Other logging configurations are not covered here. 
 
 ##### View Blocklog
-Best viewed in CNTools or gLiveView. See [CNCLI](Scripts/cncli.md) for example output.
\ No newline at end of file
+Best viewed in CNTools or gLiveView. See [CNCLI](Scripts/cncli.md) for example output.
+
+#### Telegram alerts
+
+If known but unwanted errors occur on your node, or if characteristic values indicate an unusual status , cntools can send you Telegram alert messages. 
+
+To do this, you first have to activate your own bot and link it to your own Telegram user. Here is an explanation of how this works:
+
+1. Open Telegram and search for "*botfather*"
+
+2. write him your wish: `/newbot`
+
+3. define a name for your bot, such as `cntools_[POOLNAME]_alerts`
+
+4. Botfather will confirm the creation of your bot by giving you the unique **bot access token**. Keep it save and private.
```

Comment:
LGTM, just a small typo in "Keep it **save** and private."

I can see why you placed it in logmonitor md as this is one use case for the TG alerts. But it could just as well be used in other scripts so maybe it deserves its own section in the script section of the docs?